back-up-and-restore-data.md

```diff
@@ -48,9 +48,10 @@ $ cockroach dump --help
 
 The `cockroach dump` command supports the following flags as well as [logging flags](cockroach-commands.html#logging-flags).
 
-Flag | Description 
+Flag | Description
 -----|------------
-`--ca-cert` | The path to the [CA certificate](create-security-certificates.html). This flag is required if the cluster is secure.<br><br>**Env Variable:** `COCKROACH_CA_CERT` 
+`--as-of` | Dump table schema and/or data as they appear before the specified [timestamp](timestamp.html). See this [example](#dump-table-data-as-of-a-specific-time) for a demonstraion.<br><br>**Default:** Current time
```

Comment:
Either here or in the as-of example below would be a good place to note the limitations on -as-of wrt the GC interval, so maybe a link to https://www.cockroachlabs.com/docs/select.html#select-historical-data-time-travel is useful.